Randomizer.cpp

```diff
@@ -812,35 +813,26 @@ namespace Randomizer {
             return false;
         }
 
-        if (options.race) {
-            std::cout << "Race mode enabled, randomizing seed number.\n";
-            seed = 0;
-        }
-
         seed = Random::RandomInit(seed);
         std::string seedText = std::to_string(seed);
-
-        if (options.race) {
-            seedText += std::to_string(Random::RandomInteger(INT32_MAX));
-            // Get top 10 characters of SHA1 hash
-            boost::uuids::detail::sha1 sha1;
-            sha1.process_bytes(seedText.data(), seedText.size());
-            unsigned hash[5] = {0};
-            sha1.get_digest(hash);
-            char buffer[41] = {0};
-            for (int i = 0; i < 5; i++) {
-                std::sprintf(buffer + (i << 3), "%08x", hash[i]);
-            }
-            seedText = "Race ";
-            seedText += std::string(buffer, 5);
-        }
-
         if (seedName) {
             *seedName = seedText;
         }
-
         std::cout << "Seed " << seedText << "\n";
 
+        /*
+            Race mode: we increment the seed a few times from what is actually reported.
+            This means that if you take the same seed and give it without race mode
+            the result will be different. But the result with race mode will be consistent
+            and there is a seed we can report and use as a link.
+        */
+        if (options.race) {
+            std::cout << "Race mode enabled, altering actual seed.\n";
+            seed = Random::RandomInit(seed);
+            seed = Random::RandomInit(seed);
+            seed = Random::RandomInit(seed);
```

Comment:
Assignment to `seed` is not necessary here, because that variable will not be read again and leads to the following compiler warning:

```
/tmp/tmp.gxyyoUonc6/RandoBlazer/Randomizer.cpp:833:13: warning: Value stored to 'seed' is never read [deadcode.DeadStores]
            seed = Random::RandomInit(seed);
            ^      ~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
Makes sense, done.